ci-scripts/yocto2.groovy

```diff
@@ -180,7 +180,12 @@ String getStringEnvVar(String envVarName, String defaultValue) {
     return returnValue
 }
 
-void buildManifest(String variantName, String imageName, String layerToReplace="", String newLayerPath="") {
+// "machine" is a name of a valid yocto machine.
+// Example: machine = "intel-corei7-64"
+//
+// "features" is a list of strings representing additional features in the build.
+// Example: features = ["qtauto"]
+void buildManifest(String machine, List features, String imageName, String layerToReplace="", String newLayerPath="") {
```

Comment:
Main file for Jenkins is [https://github.com/Pelagicore/pelux-manifests/blob/new_variant_ci/Jenkinsfile](Jenkinsfile) buildManifest call should be changed there as well

Reply:
good catch

Reply:
should be fixed now